web/src/click_handlers.ts

```diff
@@ -577,7 +577,12 @@ export function initialize(): void {
             $(".user_sidebar_entry .status-emoji-name").on("mouseenter", () => {
                 const element: tippy.ReferenceElement = util.the($elem);
                 const instance = element._tippy;
-                if (instance?.state.isVisible) {
+                // We make sure instance is not of user popover since we don't want to
+                // close that.
+                if (
+                    instance?.state.isVisible &&
+                    !instance.popper.classList.contains("user-sidebar-popover-root")
```

Comment:
I think we want a positive test for whether the popover is the status emoji tooltip, not a negative test for whether it's the user card popover.